### ISSUE TITLE
Error in about.xml

### ISSUE THREAD

**Issue by RelaxedOtter** [Alexis-Grimm/Un-Limited#4]
The latest version of the About.xml file has an error in the LoadAfter usage, causing red errors in the log. To fix it change it to:
```
<loadAfter>
    <li>syrchalis.harvestyieldpatch</li>
</loadAfter>
```
no need to use the <packageId> in the list element.

**Comment by Alexis-Grimm** [comment]
Thanks, I was just about to look into how the about.xml stuff worked. I'll have it fixed today.

**Comment by Alexis-Grimm** [comment]
Fixed in version 1.2.3
